js/pro/wazirx.js

```diff
@@ -0,0 +1,701 @@
+        //             "gc": 'usdt',
+        //             "m": true,
+        //             "o": 26946170,
+        //             "p": "5.0",
+        //             "q": "20.0",
+        //             "s": "btcusdt",
+        //             "t": 17376032,
+        //             "w": "100.0"
+        //         },
+        //         "stream": "ownTrade"
+        //     }
+        //
+        const trade = this.safeValue (message, 'data', {});
+        const messageHash = 'myTrades';
+        let myTrades = undefined;
+        if (this.myTrades === undefined) {
+            const limit = this.safeInteger (this.options, 'tradesLimit', 1000);
+            myTrades = new ArrayCacheBySymbolById (limit);
+            this.myTrades = myTrades;
+        } else {
+            myTrades = this.myTrades;
+        }
+        const parsedTrade = this.parseWSTrade (trade);
+        myTrades.append (parsedTrade);
+        client.resolve (myTrades, messageHash);
+    }
+
+    handleConnected (client, message) {
+        //
+        //     {
+        //         data: {
+        //             timeout_duration: 1800
+        //         },
+        //         event: 'connected'
+        //     }
+        //
+        return message;
+    }
+
+    handleSubscribed (client, message) {
+        //
+        //     {
+        //         data: {
+        //             streams: ['!ticker@arr']
+        //         },
+        //         event: 'subscribed',
+        //         id: 0
+        //     }
+        //
+        return message;
+    }
+
+    handleError (client, message) {
+        //
+        //     {
+        //         "data": {
+        //             "code": 400,
+        //             "message": "Invalid request: streams must be an array"
+        //         },
+        //         "event": "error",
+        //         "id": 0
+        //     }
+        //
+        throw new ExchangeError (this.id + ' ' + this.json (message));
+    }
+
+    handleMessage (client, message) {
+        const event = this.safeString (message, 'event');
+        const eventHandlers = {
+            'error': this.handleError,
+            'connected': this.handleConnected,
+            'subscribed': this.handleSubscribed,
+        };
+        const eventHandler = this.safeValue (eventHandlers, event);
+        if (eventHandler !== undefined) {
+            return eventHandler.call (this, client, message);
+        }
+        const stream = this.safeString (message, 'stream', '');
+        const streamHandlers = {
+            'ticker@arr': this.handleTicker,
+            '@depth': this.handleOrderBook,
+            '@kline': this.handleOHLCV,
+            'outboundAccountPosition': this.handleBalance,
+            'orderUpdate': this.handleOrder,
+            'ownTrade': this.handleMyTrades,
+        };
+        const streams = Object.keys (streamHandlers);
+        for (let i = 0; i < streams.length; i++) {
+            if (this.inArray (streams[i], stream)) {
```

Comment:
@pcriadoperez I know this is old, but do you remember what you were trying to check for here? The problem with this line is that both arguments are a string